.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```
@@ -53,4 +91,4 @@ body:
       description: By submitting this issue, you agree to follow our [Code of Conduct](https://github.com/danny-avila/LibreChat/blob/main/.github/CODE_OF_CONDUCT.md)
       options:
         - label: I agree to follow this project's Code of Conduct
-          required: true
+          required: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)